caffe2_builders/v140/auto.bat

```diff
@@ -0,0 +1,102 @@
+rem Remove to original folder after script is finished
+set ORIGINAL_DIR=%cd%
+
+rem Build folders are 
+rem %CAFFE2_ROOT%\build\Debug for Debug and
+rem %CAFFE2_ROOT%\build\Release for Release
+set CAFFE2_ROOT=%~dp0%\pytorch
+
+rem Cloning repository if it doesn't exist
+if not exist pytorch (
+    set PYTORCH_BRANCH=windows-shared-build
+    set PYTORCH_REPO=ArutyunovG
+    call  %~dp0%..\..\internal\clone.bat
+    cd %ORIGINAL_DIR%
+)
+
+rem Should build folder be deleted and build start from scratch?
+if NOT DEFINED REBUILD (
+    set REBUILD=0
+)
+
+rem Debug build enabled by default
+if NOT DEFINED BUILD_DEBUG (
+    set BUILD_DEBUG=1
+)
+
+rem Release build enabled by default
+if NOT DEFINED BUILD_RELEASE (
+    set BUILD_RELEASE=1
+)
+
+rem msbuild /m: option value
+if NOT DEFINED MAX_JOBS (
+    set MAX_JOBS=6
```

Comment:
Consider using `%NUMBER_OF_PROCESSORS%` instead?

Reply:
As far as I know, it is rather number of processes, not devices.
https://www.hanselman.com/blog/FasterBuildsWithMSBuildUsingParallelBuildsAndMulticoreCPUs.aspx
Switch it to NUMBER_OF_PROCESSES?

Reply:
%NUMBER_OF_PROCESSORS% stores the number of logical processors. For example, I have an i7-7700hq and then this env var is set to 8 by the OS. It's not good to pin the MAX_JOBS to a constant number.